test/common/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ PCL_ADD_TEST(common_bearing_angle_image test_bearing_angle_image FILES test_bear
 
 PCL_ADD_TEST(common_point_type_conversion test_common_point_type_conversion FILES test_point_type_conversion.cpp LINK_WITH pcl_gtest pcl_common)
 PCL_ADD_TEST(common_colors test_colors FILES test_colors.cpp LINK_WITH pcl_gtest pcl_common)
+PCL_ADD_TEST(common_make_shared test_make_shared FILES test_make_shared.cpp LINK_WITH pcl_gtest pcl_common)
```

Comment:
Proposal to rename the file to `test_[type_]traits`. Currently, it is named after make shared, but not a single call to `make_shared` takes place.